app/src/main/res/values/colors.xml

```diff
@@ -27,4 +27,5 @@
   <color name="blueTransparent">#962e7ac4</color>
   <color name="actionModeBackground">#4285F4</color>
   <color name="titleBar">#000000</color>
+  <color name="green_tick">#4CAF50</color>
```

Comment:
We should follow a consistent style so change this to camel case.

Reply:
@abdulwd But if you will see a few lines above, the style is snake case. What should I do? In fact, when I was naming it, I myself was confused which was to choose. Well according to material guidelines it should be camel case. Should I change others to camel case too?

Reply:
Just change yours to camel case. We can change others later.

Reply:
@abdulwd Done.